Xamarin.Forms.Platform.UAP/TabbedPageRenderer.cs

```diff
@@ -232,16 +237,48 @@ void OnLoaded(object sender, RoutedEventArgs args)
 			UpdateBarTextColor();
 			UpdateBarBackgroundColor();
 			UpdateBarIcons();
+			UpdateAccessKeys();
 		}
 
-		void OnPagesChanged(object sender, NotifyCollectionChangedEventArgs e)
-		{
+		void OnPagesChanged(object sender, NotifyCollectionChangedEventArgs e) {
 			e.Apply(Element.Children, Control.Items);
-
+			switch (e.Action)
+			{
+				case NotifyCollectionChangedAction.Add:
+				case NotifyCollectionChangedAction.Remove:
+				case NotifyCollectionChangedAction.Replace:
+					if (e.NewItems != null)
+						foreach (var c in e.NewItems)
+							(c as Page).PropertyChanged += OnChildPagePropertyChanged;
```

Comment:
I wonder if there is a way to not sub/unsub this event and still achieve this effect. Its expensive to sub PropertyChanged as it triggers a fair bit...

Reply:
Good point. However - is it that big difference compared to other property change handling.
One thought could be to try to handle this in xaml with converters or something? Haven't looked alot into that yet though.